mkdocs/docs/tutorials/gitops/argocd.md

```diff
@@ -0,0 +1,211 @@
+
+You can now merge the PR that Iter8 just created. Argo CD should detect the change and sync the cluster to the new desired states. If the experiment had succeeded, the candidate version will become the new baseline version for future experiments.
+
+??? info "More about Iter8 GitOps handler task"
+    Iter8 operating in the GitOps mode is very similar to how it works normally. One key difference is that at the end of the experiment, it will need to perform an additional step to modify the desired state of the Env repo to reflect the outcome of the experiment. This can be done by specifying a `finish` Iter8 handler task that runs at the end of an experiment. The specific handler task we are using in this tutorial is written in a shell script as shown below:
+    ```yaml linenums="1"
+    actions:
+      # when the experiment completes, promote the winning version in the Env repo
+      finish:
+      - task: common/exec
+        with:
+          cmd: /bin/bash
+          args: [
+                 "-c",
+                 "apt-get install -y git jq curl;\
+                  REPO=github.com/huang195/iter8;\
+                  BRANCH=gitops;\
+                  USER=huang195;\
+                  TOKEN=`kubectl -n {{ .namespace }} get secret iter8-token -o json | jq -r .data.token | base64 -d`;\
+                  git config --global user.email 'iter8@iter8.tools';\
+                  git config --global user.name 'Iter8';\
+                  git clone https://${USER}:${TOKEN}@${REPO} --branch=${BRANCH};\
+                  cd iter8/samples/gitops;\
+                  TMP=`mktemp`;\
+                  sed 's/candidate/stable/g' {{ .filepath }} > $TMP;\
+                  mv $TMP productpage.yaml;\
+                  rm -f productpage-candidate.yaml;\
+                  rm -f fortio.yaml;\
+                  rm -f experiment.yaml;\
+                  if [ `git status -s | wc -l` != 0 ];\
+                  then \
+                    git checkout -b iter8_exp;\
+                    git commit -a -m 'update baseline';\
+                    git push -f origin iter8_exp;\
+                    curl -u${USER}:${TOKEN} -XPOST https://api.github.com/repos/iter8-tools/iter8/pulls -s -d '{\"head\":\"iter8_exp\", \"base\":\"gitops\", \"body\":\"update baseline\", \"title\":\"Iter8 GitOps\"}';\
+                  fi;\
+                  kubectl -n {{ .namespace }} apply -f istio-vs.yaml;\
+                 "
+                ]
+
+    ```
+
+    For prototyping, one can write these handler tasks as shell scripts and inline them within an Experiment CR. This makes writing these handlers extremely efficient and easy to debug. However, the down side is it makes the Experiment CR a lot more complicated and scary to read. We are currently working to simplify the handler interface, so stay tuned.
```

Comment:
This is a style comment.... perhaps we could remove `extremely` from the two sentences in which they occur?